src/Component.js

```diff
@@ -1,9 +1,9 @@
-import associateInstance from './associateInstance';
-import { reRender } from './render';
-import { mergeState } from './utils';
+import associateInstance from "./associateInstance";
```

Comment:
Single quote for imports. I think you have enabled prettier. Can you disable prettier and make the changes. I will see if I can add prettier config so it doesn't prettifies unexpectedly.